pkg/virt-controller/services/template.go

```diff
@@ -1548,17 +1548,6 @@ func renderLaunchManifestVolumes(t *templateService, vmi *v1.VirtualMachineInsta
 			}
 		}
 
-		if util.HasHugePages(vmi) && vmi.Spec.Domain.Memory != nil && vmi.Spec.Domain.Memory.Guest != nil {
-			volumes = append(volumes, k8sv1.Volume{
-				Name: volume.Name,
-				VolumeSource: k8sv1.VolumeSource{
-					PersistentVolumeClaim: &k8sv1.PersistentVolumeClaimVolumeSource{
-						ClaimName: volume.PersistentVolumeClaim.ClaimName,
-						ReadOnly:  volume.PersistentVolumeClaim.ReadOnly,
-					},
-				},
-			})
-		}
```

Comment:
This code was introduced in https://github.com/kubevirt/kubevirt/pull/6118/commits/3ff727b552e3c6d86aceba6eddf37f1765439c90. 

Please squash this change there.

Reply:
done